File: test_gemini.js
const { GoogleGenerativeAI } = require('@google/generative-ai');

const API_KEY = "AIzaSyC0wxMtsDtvmSv5N1WaGquQkpyECxp7T6c";

async function testGemini() {
  console.log('🔑 Testing Gemini API...');
  
  try {
    const genAI = new GoogleGenerativeAI(API_KEY);
    
    console.log('📦 Creating model: gemini-2.0-flash-exp');
    const model = genAI.getGenerativeModel({ 
      model: 'gemini-2.0-flash-exp',
      generationConfig: {
        temperature: 0.9,
        topK: 40,
        topP: 0.95,
        maxOutputTokens: 2048,
      }
    });
    
    console.log('💬 Sending message: "Hello, who are you?"');
    const result = await model.generateContent("Hello, who are you?");
    const response = await result.response;
    const text = response.text();
    
    console.log('✅ SUCCESS!');
    console.log('📝 Response:', text);
    
  } catch (error) {
    console.error('❌ ERROR:', error.message);
    console.error('Full error:', error);
  }
}

testGemini();

File: test_gemini.py
import google.generativeai as genai

# Your API key
API_KEY = "AIzaSyC0wxMtsDtvmSv5N1WaGquQkpyECxp7T6c"

# Configure the API
genai.configure(api_key=API_KEY)

# Test with Gemini 2.0 Flash
print("Testing Gemini 2.0 Flash...")
try:
    model = genai.GenerativeModel('gemini-2.0-flash-exp')
    response = model.generate_content("Hello, how are you?")
    print("✅ Success!")
    print(f"Response: {response.text}")
except Exception as e:
    print(f"❌ Error: {e}")
